gitee/payload.go

```diff
@@ -0,0 +1,289 @@
+package gitee
+
+import (
+	"time"
+)
+
+type CommentEventPayload struct {
+	Action        *string          `json:"action,omitempty"`
```

Comment:
Just to be sure, do we actually need use pointers everywhere?
Probably sometimes it make sense to have simple types as is. Or you should distinguish empty string and nil?

Reply:
BTW, you can generally omit "omitempty" for "incoming" types. You will use full structure anyway. It's more make sense for instantiating JSONs.

Reply:
> BTW, you can generally omit "omitempty" for "incoming" types. You will use full structure anyway. It's more make sense for instantiating JSONs.

Thanks, I'll make some improvements and then retest it.